### PR TITLE
Compatible Page Updates

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR updates the compatibility page.
💻 [Preview](https://deploy-preview-1134--docs-spectrocloud.netlify.app/component)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1134#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/16-component.md
Comment (ritawatson) [inline content/docs/16-component.md:59]:
```suggestion
|Palette Release|On-Prem Installer Version|
```

Comment (ritawatson) [reply content/docs/16-component.md:59]: Per our word list, change On-Premises to On-Prem in section title and table header.

File: content/docs/16-component.md
Comment (ritawatson) [inline content/docs/16-component.md:84]: Should PCG images be grouped under a heading 1 as follows? That categorizes and allows spelling out PCG.

# Private Cloud Gateway Images
## vSphere PCG Image Version
## MAAS PCG Image Version
## OpenStack PCG Image Version

File: content/docs/16-component.md
Comment (ritawatson) [inline content/docs/16-component.md:176]:
```suggestion
- [Downloads](/spectro-downloads#palettedynamicartifacts)
```

**Review by ritawatson — APPROVED** [review]
A couple minor suggestions: re headings for PCG images and remove one period at the end.
LGTM

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1134#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1134#pullrequestreview-1302158375" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
